InterfacePython/TasmanianSG.py

```diff
@@ -1525,6 +1525,13 @@ def setAnisotropicRefinement(self, sType, iMinGrowth, iOutput, liLevelLimits = [
             sType = bytes(sType, encoding='utf8')
         pLibTSG.tsgSetAnisotropicRefinement(self.pGrid, c_char_p(sType), iMinGrowth, iOutput, pLevelLimits)
 
+    def getAnisotropicRefinement(self, sType, iMinGrowth, iOutput, liLevelLimits = []):
+        '''
+        Calls setAnistropicRefinement() on the inputs and then getNeededPoints().
+        '''
+        self.setAnisotropicRefinement(sType, iMinGrowth, iOutput, liLevelLimits = [])
```

Comment:
What?! That's now how you pass in parameters in Python.

Also, have them implemented in also in C++.

Reply:
Fixed. Will do the C++ functions now.